docs/remote_inference_blueprints/amazon_rekognition_connector_blueprint.md

```diff
@@ -0,0 +1,259 @@
+# Amazon Rekognition Detect Text connector blueprint
+
+Amazon Rekognition is a computer vision service that can analyze images and videos. This tutorial shows how to create an OpenSearch connector for Amazon Rekognition's Detect Text API, which can detect and extract text from images.
+
+## Detect text with DetectText API
+
+This tutorial demonstrates how to create an OpenSearch connector for an Amazon Rekognition model that detects text in images using the DetectText API.
+
+Note: This functionality is available in OpenSearch 2.14 or later.
+
+### Step 1: Add the connector endpoint to trusted URLs
+```json
+PUT /_cluster/settings
+{
+  "persistent": {
+    "plugins.ml_commons.trusted_connector_endpoints_regex": [
+      "^https://rekognition\\..*[a-z0-9-]\\.amazonaws\\.com$"
+    ]
+  }
+}
+```
+
+
+### Step 2: Create a connector for Amazon Rekognition
+```json
+POST /_plugins/_ml/connectors/_create
+{
+  "name": "rekognition: detect text",
+  "description": "my test connector",
+  "version": "1.0",
+  "protocol": "aws_sigv4",
+  "credential": {
+    "access_key": "your_access_key",
+    "secret_key": "your_secret_key",
+    "session_token": "your_session_token"
+  },
+  "parameters": {
+    "region": "us-west-2",
+    "service_name": "rekognition"
+  },
+  "actions": [
+    {
+      "action_type": "predict",
+      "method": "POST",
+      "headers": {
+        "content-type": "application/x-amz-json-1.1",
+        "X-Amz-Target": "RekognitionService.DetectText"
+      },
+      "url": "https://${parameters.service_name}.${parameters.region}.amazonaws.com",
+      "request_body": "{ \"Image\": { \"Bytes\": \"${parameters.image_bytes}\" }  } "
+    }
+  ]
+}
+``` 
+Sample response:
+```json
+{
+    "connector_id": "o52l5pMB6Ebhud5_ypxu"
+}
+``` 
+
+### Step 3: Register the model
+```json
+POST /_plugins/_ml/models/_register
+{
+    "name": "Amazon Rekoginition detect text model",
+    "version": "1.0",
+    "function_name": "remote",
+    "description": "test model",
+    "connector_id": "o52l5pMB6Ebhud5_ypxu"
+}
+``` 
+Sample response:
+```json
+{
+    "task_id": "pZ2n5pMB6Ebhud5_n5yK",
+    "status": "CREATED",
+    "model_id": "pp2n5pMB6Ebhud5_oJwF"
+}
+``` 
+
+### Step 4: Deploy the model
+```json
+POST /_plugins/_ml/models/pp2n5pMB6Ebhud5_oJwF/_deploy
+``` 
+Sample response:
+```json
+{
+    "task_id": "p52o5pMB6Ebhud5_dpyL",
+    "task_type": "DEPLOY_MODEL",
+    "status": "COMPLETED"
+}
+``` 
+
+### Step 5: Test the model inference
+```json
+POST _plugins/_ml/models/pp2n5pMB6Ebhud5_oJwF/_predict
+{
+  "parameters": {
+    "response_filter": "$.TextDetections.*.DetectedText",
```

Comment:
Suggest not add `response_filter` to show the raw model response. Then introduce how to use `response_filter` to filter target parts.